app/screens/Login/index.js

```diff
@@ -13,6 +14,19 @@ class LoginContainer extends Component {
     error: null
   };
 
+  navListener;
+
+  componentDidMount() {
+    const { addListener } = this.props.navigation;
+    addListener('didFocus', () => {
+      StatusBar.setBackgroundColor('#000000');
+    });
+  }
+
+  componentWillUnmount() {
+    this.navListener.remove();
```

Comment:
this is not a prop

Reply:
As mentioned above, i want to remove listener intance when login will unmount.